test_scripts/UpdateVideoStreamingCapabilities/001_12_GetCapabilities_caching_of_addVSC_invalid.lua

```diff
@@ -0,0 +1,79 @@
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0296-Update-video-streaming-capabilities-during-ignition-cycle.md
+--
+-- Description: SDL does not persist the videoStreamingCapability received from HMI
+--  in UI.GetCapabilities response in case HMI responds with videoStreamingCapability
+--  which contains additionalVideoStreamingCapabilities array with incorrect parameters
+--
+-- Preconditions:
+-- 1. HMICapabilitiesCacheFile is set in smartDeviceLink.ini
+-- 2. SDL and HMI are started
+--
+-- Sequence:
+-- 1. SDL requests UI.GetCapabilities()
+--   HMI sends UI.GetCapabilities(videoStreamingCapability) response with additionalVideoStreamingCapabilities
+--   which contains additionalVideoStreamingCapabilities array with incorrect parameters
+-- SDL does:
+-- - a. not cache the videoStreamingCapability with additionalVideoStreamingCapabilities
+-- 2. It is restarted ignition cycle
+-- SDL does:
+-- - a. requests UI.GetCapabilities()
+-- 3. App registers with 5 transport protocol
+--   App requests GetSystemCapability(VIDEO_STREAMING)
+-- SDL does:
+-- - a. send GetSystemCapability response with videoStreamingCapability that contains
+--    the additionalVideoStreamingCapabilities received from HMI in UI.GetCapabilities response in last ignition cycle
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local isSubscribe = false
+local appSessionId = 1
+
+local checks = { }
+
+checks.invalid_type = common.getVideoStreamingCapability(1)
+checks.invalid_type.additionalVideoStreamingCapabilities[1].supportedFormats[1].pixelPerInch = "264" -- invalid type
+
+checks.invalid_value = common.getVideoStreamingCapability(2)
+checks.invalid_value.additionalVideoStreamingCapabilities[1].preferredResolution.resolutionHeight = -2 -- invalid value
+
+checks.invalid_nested_type = common.getVideoStreamingCapability(2)
+checks.invalid_nested_type.additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(1)
+checks.invalid_nested_type.additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[1]
+  .diagonalScreenSize = true -- invalid type
+
+checks.invalid_nested_value = common.getVideoStreamingCapability(3)
+checks.invalid_nested_value.additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(2)
+checks.invalid_nested_value.additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[2]
+  .supportedFormats = "RTP" -- invalid value
```

Comment:
@aderiabin this is not the invalid value6 this is the unknown enum value and it will be erased

Reply:
Fixed (see 2a748f4)